ps_checkout.php

```diff
@@ -116,6 +116,7 @@ class Ps_checkout extends PaymentModule
         'PS_CHECKOUT_INTEGRATION_DATE' => self::INTEGRATION_DATE,
         'PS_CHECKOUT_WEBHOOK_SECRET' => '',
         'PS_CHECKOUT_LIABILITY_SHIFT_REQ' => '1',
+        'PS_CHECKOUT_PAYPAL_ENVIRONMENT' => PsCheckoutCart::PAYPAL_ENVIRONMENT_PROD_LIVE,
```

Comment:
You can replace this with constant from https://github.com/PrestaShopCorp/ps_checkout/blob/v8.3.3.1/src/PayPal/Mode.php